rest/client/methods.go

```diff
@@ -525,6 +529,40 @@ func (c *communicatorImpl) SendTestLog(ctx context.Context, taskData TaskData, l
 		return "", errors.New(message)
 	}
 	logID := logReply.ID
+
+	// Send to buildlogger.
+	tk, err := c.GetTask(ctx, taskData)
+	if err != nil {
+		return logID, errors.Wrapf(err, "problem getting task info for test %s", log.Name)
+	}
+
+	if err = c.createCedarGRPCConn(ctx); err != nil {
+		return logID, errors.Wrapf(err, "problem setting up cedar grpc connection for test %s", log.Name)
+	}
+
+	timberOpts := &buildlogger.LoggerOptions{
+		Project:    tk.Project,
+		Version:    tk.Version,
+		Variant:    tk.BuildVariant,
+		TaskName:   tk.DisplayName,
+		TaskID:     tk.Id,
+		Execution:  int32(tk.Execution),
+		TestName:   log.Name,
```

Comment:
Are we going to use test name to distinguish regular agent/system/task logs from test logs?

Reply:
Yeah, it would require changing the current route used by evergreen to fetch those logs. We would have to search by testname where the test name is empty. It would be a one line change.